test/web-platform-tests/to-run.yaml

```diff
@@ -287,6 +287,15 @@ xml-serialization.xhtml: [fail, Unknown]
 
 ---
 
+DIR: hr-time
+
+idlharness.html: [fail, ReferenceError - WebIDL2 is not defined]
```

Comment:
One day we should figure out what this is about. I guess it's probably script execution stuff :(

Reply:
I had a quick look and the cause seems to be: https://github.com/w3c/web-platform-tests/issues/5608

The redirect for `/resources/WebIDLParser.js` to `/resources/webidl2/lib/webidl2.js` doesn't seem to occur with the way we launch the WPT server, as the test works when I use the actual path.

Maybe it can be resolved by modifying `wpt-config.json`?

Reply:
I guess it wouldn't work anyway since these tests use `fetch()`, so not worth bothering with for now.